lib/pbench/test/unit/server/test_requests.py

```diff
@@ -406,10 +409,12 @@ def test_upload(
         assert dataset.controller == self.controller
         assert dataset.name == datafile.name[:-7]
         assert dataset.state == States.UPLOADED
-        assert dataset.created == dateutil.parser.parse("2002-05-16")
-        assert dataset.uploaded == dateutil.parser.parse("1970-01-01")
+        assert dataset.created == date_parser.parse("2002-05-16T00:00:00+00:00")
+        assert dataset.uploaded == date_parser.parse("1970-01-01T00:00:00+00:00")
         assert Metadata.getvalue(dataset, "dashboard") is None
-        assert Metadata.getvalue(dataset, Metadata.DELETION) == "1972-01-01"
+        assert (
+            Metadata.getvalue(dataset, Metadata.DELETION) == "1972-01-01T00:00:00+00:00"
+        )
```

Comment:
Is there a reason why the `dataset` `created` and `uploaded` timestamps are compared as `datetime` objects while the `DELETION` value is compared as a string?  (E.g., should they all be compared in the same way?  Might there be an advantage or disadvantage to comparing them one way instead of the other?)

Reply:
I guess what I'm wondering is whether the type of the `created` and `uploaded` timestamps is important, or whether we are ultimately most concerned with their values, particularly their values as they will be rendered at the interface.  So, it struck me that we would prefer to do the checks as strings...but, it's not a big deal.  (As far as `DELETION` goes, if it is already stored as a string, then there's no question, there.)